src/components/events/PointsDisplay/index.tsx

```diff
@@ -0,0 +1,17 @@
+import styles from './style.module.scss';
+
+interface PointsDisplayProps {
+  points: number;
+  attended: boolean;
+}
+
+const PointsDisplay = ({ points, attended }: PointsDisplayProps) => {
+  return (
+    <div className={`${styles.circle} ${attended ? styles.green : styles.blue}`}>
```

Comment:
I'd explore alternate ways to show whether you've attended an event (unless we want to keep it hidden?). Green vs blue isn't exactly clear, and relying on color isn't great for colorblind people. One idea is to show a check icon if you've attended an event

Reply:
I think we should iterate a little with design before settling here, I like the idea of a check but it does make the point total unreadable 🤔

Reply:
Maybe for now we can keep green and update in a future PR once we've figured out a direction on this